0 database

```diff
@@ -0,0 +1,2 @@
+create database trackgermdb
```

Comment:
missing like everything.... what is going on

Reply:
I was busy. will get to it right away

Reply:
I added the germ table. pleae let me know if that is sufficient